app/cmd/add_replica.go

```diff
@@ -92,11 +92,12 @@ func addReplica(c *cli.Context) error {
 
 	fastSync := c.Bool("fast-sync")
 	fileSyncHTTPClientTimeout := c.Int("file-sync-http-client-timeout")
+	grpcTimeoutSeconds := c.Int64("grpc-timeout-seconds")
```

Comment:
Need to add this arg to the CLI common flags.

Reply:
Thank you. Fixed